### PR TITLE
Update end of message history copy on mobile

### PR DESCRIPTION
**Pull request by dharit-tan**
### Description
Minor updates

### Dragons

*Is there anything the reviewer should be on the lookout for? Are there any dangerous changes?*

### How Has This Been Tested?

local ios stage

### How will this change be monitored?

*For features that are critical or could fail silently please describe the monitoring/alerting being added.*

### Feature Flags ###

*Are all new features properly feature flagged? Describe added feature flags.*

### CODE REVIEW

File: packages/mobile/src/screens/chat-screen/ChatScreen.tsx [hunk 1]
```diff
@@ -478,7 +472,7 @@ export const ChatScreen = () => {
   }, [])
 
   const handleOnContentSizeChanged = useCallback(
-    (_contentWidth, contentHeight) => {
+    (contentWidth, contentHeight) => {
```
Comment (rickyrombo) [inline packages/mobile/src/screens/chat-screen/ChatScreen.tsx:475]: won't this break linter?

**Review by rickyrombo — APPROVED** [review]

File: packages/mobile/src/screens/chat-screen/ChatScreen.tsx (hunk 1 above)
Comment (dharit-tan) [reply packages/mobile/src/screens/chat-screen/ChatScreen.tsx:475]: seems like it passed.. i've had unused vars without the _ before.

Comment (rickyrombo) [reply packages/mobile/src/screens/chat-screen/ChatScreen.tsx:475]: seems like a bug with our linter 🤷

Comment (dharit-tan) [reply packages/mobile/src/screens/chat-screen/ChatScreen.tsx:475]: ha fair

**Pull request merged**